### PR TITLE
Space Travelers Hub

### PR DESCRIPTION
**Pull request by mwenyoa**
### In this project, we did the following to achieve a working single page webpage for reserving rockets & missions:

- Install [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).
 - Download the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
 - Create routes and view components (Rockets, Missions, My Profile, We leveraged for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).
 - Create directories for all Redux state slice files (rockets, & missions).
-  Fetch [rockets](https://api.spacexdata.com/v3/rockets) & [missions](https://api.spacexdata.com/v3/missions) data.
-  Employed the use of useselector, createAsynctrunk and other react dependencies.
 - Set up testing libraries(React Testing Library and Jest).
- Tested the Application  using React Testing Library and Jest

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <header className="d-flex">
+    <div className="d-flex logo-title">
+      <img className="logo" src={logo} alt="spacex logo" />
+      <h1><b>Space Travelers&apos; Hub</b></h1>
+    </div>
+    <nav className="d-flex nav-links">
+      <NavLink to="/">Rockets</NavLink>
+      <NavLink to="/Missions">Missions</NavLink>
+      <div className="line" />
+      <NavLink to="/Profile">My Profile</NavLink>
```
Comment (Deepakdanger) [inline src/components/Header.js]: - [ ] As per requirement, the active page should be indicated with an underline and all the page links.

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+### Install
+
+- Install npm: 
+ `npm install`
+
+### Usage
+
+- To get project open and running in your browser run `npm start` command which will start react application.
+
+
```
Comment (Deepakdanger) [inline README.md]: - [x] I would recommend adding the command to test the application as well.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @mwenyoa & @tugsem, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required changes
_Check the comments under the review._


## Optional changes
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Header.js (hunk 1 above)
Comment (mwenyoa) [reply src/components/Header.js]: fixed eror

I have fixed it now. thanks for your kind feedback

**Review by julie-ify — APPROVED** [review]
Hi @mwenyoa and @tugsem,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

# Status: Approved ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**